app/controllers/contact_controller.rb

```diff
@@ -11,9 +11,24 @@ def question
     end
   end
 
+  def suggestion
+    @form = SuggestCorrectionForm.new(suggestion_params)
+    if @form.valid? && @form.submit
+      flash[:success] = 'Your suggestion has been submitted'
```

Comment:
I think we may want to change this to flash.now for all of these, but we can wait and do that in a separate commit on all 3